.env

```diff
@@ -0,0 +1,13 @@
+OH_MARIADB_SERVER=database
+OH_MARIADB_PORT=3306
+OH_MARIADB_DATABASE=oh
+OH_MARIADB_ROOT_PASSWORD=root
+OH_MARIADB_USER=isf
+OH_MARIADB_PASSWORD=isf123
+OH_DEMO_LANG=en
+
+OH_CORE_BRANCH=develop
+OH_API_BRANCH=develop
+GITHUB_ORG=intesys
```

Comment:
Is this the correct repo?   And shouldn't there be some Java code here too?

Reply:
I don't think this PR has any meaning as for now